OWNERS

```diff
@@ -0,0 +1,33 @@
+# This file is required in order to control `/approve` and `/lgtm` and related
+# Prow commands on this repo.  It may contain non-Openshift org members and
+# aliases (defined in OWNERS_ALIASES). It will be synced to the
+# `openshift/release/ci-operator` repo where non-Openshift org members will be
+# filtered out and aliases will be expanded.
+
+# See: https://coreos.slack.com/archives/CBN38N3MW/p1654872921259859
```

Comment:
Slack links will rot out.  If you have something relevant from this discussion, probably just inline it here.